lib/remote_helper.py

```diff
@@ -291,6 +290,39 @@ def is_port_open(cls, host, port, is_verbose):
       AppScaleLogger.verbose(str(exception), is_verbose)
       return False
 
+  @classmethod
+  def merge_authorized_keys(cls, host, keyname, user, is_verbose):
+    """ Adds the contents of the user's authorized_keys file to the root's
+    authorized_keys file.
+
+    Args:
+      host: A str representing the host to enable root logins on.
+      keyname: A str representing the name of the SSH keypair to login with.
+      user: A str representing the name of the user to login as.
+      is_verbose: A bool indicating if we should print the command we execute to
+        enable root login to stdout.
+    """
+    AppScaleLogger.log('Root login not enabled - enabling it now.')
+
+    create_root_keys = 'sudo touch /root/.ssh/authorized_keys'
```

Comment:
Sounds good.
Please add a "Raises" section to the method docstring to make that clear.